lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -65,9 +75,67 @@ def _get_config(self, config_dict):
                                config_dict['ovs_internal_sampling_port_number'],
             internal_sampling_fwd_tbl=
                               config_dict['ovs_internal_sampling_fwd_tbl_number'],
+            multi_tunnel_flag=multi_tunnel_flag,
 
         )
 
+    def _ovs_multi_tunnel_init(self):
+        try:
+            subprocess.check_output("sudo ovs-vsctl list Open_vSwitch | grep gtpu",
+                                     shell=True,)
+            self.ovs_gtp_type = "gtpu"
+        except subprocess.CalledProcessError:
+            self.ovs_gtp_type = "gtp"
+
+        try:
+            subprocess.check_output('ovs-vsctl set-manager %s' % OVSDB_MANAGER_ADDR,
```

Comment:
Why are you seeing the manager?

Reply:
RYU controller is used for ovsdb address for init schema in vsctl.
Without set the set-manager with ovsdb, below error is getting during execution the use-case:

Feb  8 09:41:32 magma-dev pipelined[11861]:   File "/home/vagrant/magma/lte/gateway/python/magma/pipelined/utils.py", line 68, in get_ovs_bridge
Feb  8 09:41:32 magma-dev pipelined[11861]:     cls.ovs.init()
Feb  8 09:41:32 magma-dev pipelined[11861]:   File "/home/vagrant/build/python/lib/python3.5/site-packages/ryu/lib/ovs/bridge.py", line 150, in init
Feb  8 09:41:32 magma-dev pipelined[11861]:     self.br_name = self._get_bridge_name()
Feb  8 09:41:32 magma-dev pipelined[11861]:   File "/home/vagrant/build/python/lib/python3.5/site-packages/ryu/lib/ovs/bridge.py", line 158, in _get_bridge_name
Feb  8 09:41:32 magma-dev pipelined[11861]:     self.run_command([command])
Feb  8 09:41:32 magma-dev pipelined[11861]:   File "/home/vagrant/build/python/lib/python3.5/site-packages/ryu/lib/ovs/bridge.py", line 137, in run_command
Feb  8 09:41:32 magma-dev pipelined[11861]:     self.vsctl.run_command(commands, self.timeout, self.exception)
Feb  8 09:41:32 magma-dev pipelined[11861]:   File "/home/vagrant/build/python/lib/python3.5/site-packages/ryu/lib/ovs/vsctl.py", line 1295, in run_command
Feb  8 09:41:32 magma-dev pipelined[11861]:     self._run_command(commands)
Feb  8 09:41:32 magma-dev pipelined[11861]:   File "/home/vagrant/build/python/lib/python3.5/site-packages/ryu/lib/ovs/vsctl.py", line 1275, in _run_command
Feb  8 09:41:32 magma-dev pipelined[11861]:     self._do_main(commands)
Feb  8 09:41:32 magma-dev pipelined[11861]:   File "/home/vagrant/build/python/lib/python3.5/site-packages/ryu/lib/ovs/vsctl.py", line 1172, in _do_main
Feb  8 09:41:32 magma-dev pipelined[11861]:     self._init_schema_helper()
Feb  8 09:41:32 magma-dev pipelined[11861]:   File "/home/vagrant/build/python/lib/python3.5/site-packages/ryu/lib/ovs/vsctl.py", line 1059, in _init_schema_helper
Feb  8 09:41:32 magma-dev pipelined[11861]:     vswitch_idl.OVSREC_DB_NAME)
Feb  8 09:41:32 magma-dev pipelined[11861]:   File "/home/vagrant/build/python/lib/python3.5/site-packages/ryu/lib/ovs/vsctl.py", line 1044, in _rpc_get_schema_json
Feb  8 09:41:32 magma-dev pipelined[11861]:     vsctl_fatal('error %s' % os.strerror(error))
Feb  8 09:41:32 magma-dev pipelined[11861]:   File "/home/vagrant/build/python/lib/python3.5/site-packages/ryu/lib/ovs/vsctl.py", line 180, in vsctl_fatal
Feb  8 09:41:32 magma-dev pipelined[11861]:     raise Exception(msg)        # not call ovs.utils.ovs_fatal for reusability
Feb  8 09:41:32 magma-dev pipelined[11861]: Exception: error Connection refused

Reply:
ok.
Lets move this to 'lte/gateway/roles/magma/files/systemd/magma_pipelined.service'

Reply:
Changes done.